backend/typescript/graphql/resolvers/adminResolvers.ts

```diff
@@ -0,0 +1,43 @@
+import AdminService from "../../services/implementations/adminService";
+import {
+  IAdminService,
+  NotificationDTO,
+} from "../../services/interfaces/adminService";
+
+// change staff_id to StaffService when its done
+const adminService: IAdminService = new AdminService(3);
+
+const adminResolvers = {
+  Query: {
+    notificationById: async (
+      _parent: undefined,
+      { id }: { id: number },
+    ): Promise<NotificationDTO> => {
+      return adminService.getNotificationById(id);
```

Comment:
Correct me if I'm wrong, but I think there should be a await right after the return (same with line 19)

Reply:
linting suggested not to add this